modules/wizard/view-wizard-page.php

```diff
@@ -46,10 +56,10 @@
 		</h1>
 		<ol class="pll-wizard-steps">
 			<?php
-			foreach ( $this->steps as $step_key => $step ) {
-				$is_completed = array_search( $this->step, array_keys( $this->steps ), true ) > array_search( $step_key, array_keys( $this->steps ), true );
+			foreach ( $steps as $step_key => $step ) {
+				$is_completed = array_search( $step, array_keys( $steps ), true ) > array_search( $step_key, array_keys( $steps ), true );
```

Comment:
`$this->step` is not `$step`:
```suggestion
				$is_completed = array_search( $current_step, array_keys( $steps ), true ) > array_search( $step_key, array_keys( $steps ), true );
```